README.md

```diff
@@ -256,6 +258,29 @@ flow.status
 
 ## Advanced features
 
+### Global parameters for jobs
+
+Workflows can accept a hash of `globals` that are automatically forwarded as parameters to all jobs.
+
+This is useful to have common functionality across workflow and job classes, such as tracking the creator id for all instances:
+
+```ruby
+class SimpleWorkflow < Gush::Workflow
```

Comment:
This may not be the best name for the example as it could be confused with AWS Simple Workflow (SWF)

Reply:
`SimpleWorkflow` is already in use in Gush documentation.